tests/test_auth.c

```diff
@@ -68,73 +68,6 @@ server_thread(void *arg)
     return NULL;
 }
 
-static char *
```

Comment:
PAM can be still tested even without the `confdir` support with [`pam_wrapper`](https://cwrap.org/pam_wrapper.html). Here's how we're doing that in another CMake-based project:

https://github.com/CESNET/rousette/blob/master/CMakeLists.txt#L171

One of the gotchas is that `pam_wrapper` performs some serious magic in the background which involves per-process temporary `.so` files as well as binary patching that `libpam.so` on disk. That's why we [ended up running it in a mount namespace with a separate `/tmp`](https://github.com/CESNET/rousette/commit/9909cad66c731d8282720730e4f8fbc78fdf83bb). Previously, we used `pam_userdb`, but that one will be undergoing some important changes in the next release of modern distros, so we [ported away from that](https://github.com/CESNET/rousette/commit/248e5aa0916313a048cbc4fad644be6c52dc64df).

Reply:
Thank you, I will take a look at this. I wasn't happy about removing the PAM module and the testing of the kbint method, but I thought it wasn't possible.